_posts/2017-08-23-RTL.md

```diff
@@ -19,4 +19,9 @@ Add following Custom CSS:
     direction: rtl;
 }
 ```
-
+To exclude code blocks from the RTL effect add the following Custom CSS:
+```css
+[class^="Code"] {
```

Comment:
I think we use lower case classes, so maybe `[class^="code"]` is better

Reply:
Thank you for your response. I fixed it as you suggested.

Reply:
Sorry I just checked and it does not work properly if it is lower case.

Reply:
I see, `class^="Code"` will match `Codemirror` 
But I think 
```
pre, code {
direction: ltr;
}
```
is enough, both code blocks and inline code should be LTR, right?